src/install.coffee

```diff
@@ -85,7 +84,8 @@ class Install extends Command
     opts = {env, cwd: @atomDirectory}
     opts.streaming = true if @verbose
 
-    @fork @atomNodeGypPath, installNodeArgs, opts, (code, stderr='', stdout='') ->
+    atomNodeGypPath = process.env.ATOM_NODE_GYP_PATH or require.resolve('node-gyp/bin/node-gyp')
```

Comment:
This feels a bit weird to me. I poked around a bit trying to use `@npm.config.get('node-gyp')` but that's pointed to the `node_modules/.bin/` symlink, not the `node-gyp.js` script, so `@fork` wasn't happy with it, and I didn't want to swap it out for a `@spawn` because it looked important to preserve the node executable.